package.json

```diff
@@ -161,6 +160,7 @@
     "run-sequence": "^1.2.2",
     "saucelabs-connector": "^0.2.0",
     "serve-static": "^1.10.0",
+    "stack-chain": "^2.0.0",
```

Comment:
Is this dependency necessary?

Reply:
It's left in [tests](https://github.com/AndreyBelym/testcafe/blob/6e70243a736f174c36cd89d6f2abac2de73e3ee8/test/server/data/test-callsite/index.js) to compare callsites produced with `stack-chain` and without it.